index.html

```diff
@@ -42,5 +42,238 @@ <h2 class="supporting_text">
             </ul>
         <div class="headline_bg_img"></div>
     </section>
```

Comment:
The issue with the background image is till pending. I can see you made an effort to fix it but there is an empty white space on the right when you stretch it as shown below. Kindly check and fix it.

![Screenshot from 2022-09-24 01-31-19](https://user-images.githubusercontent.com/65233860/192074632-f213fcbe-e1d7-47f3-b0d6-4f07d7b58435.png)